# Starts a trajectory on the robot

int32 robotid
Trajectory trajectory
---
